test/tests/graphiql.test.js

```diff
@@ -0,0 +1,43 @@
+describe('graphql - GraphiQL', () => {
+  const cds = require('@sap/cds/lib')
+  const path = require('path')
+  const { gql } = require('../util')
+
+  const { axios, GET } = cds.test(path.join(__dirname, '../resources/bookshop-graphql'))
+  // Prevent axios from throwing errors for non 2xx status codes
+  axios.defaults.validateStatus = false
+
+  test('GET request to endpoint should serve HTML containing GraphiQL', async () => {
+    const response = await GET('/graphql')
+    expect(response.data).toMatch(/<html>[\s\S]*graphiql[\s\S]*<\/html>/)
```

Comment:
Isn't that a lengthy synonym for
```suggestion
    expect(response.data).toMatch(/<html>.*graphiql.*<\/html>/)
```

Reply:
No, I tried that at first, but unfortunately `.` doesn't match newline characters in JavaScript

Reply:
Learned something today 😄